ci/jenkins/test-vmc.sh

```diff
@@ -483,14 +483,14 @@ function run_integration {
       # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
       ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "PATH=$PATH:/usr/local/go/bin && umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && cd multicluster && NO_LOCAL=true make test-integration"
       if [[ "$COVERAGE" == true ]]; then
-        run_codecov "mc-integration-tests" "antrea/multicluster/.coverage/coverage-integration.txt" "" true ${VM_IP}
+        run_codecov "mc-integration-tests" "coverage-integration.txt" "" true ${VM_IP}
```

Comment:
how does codecov discover `coverage-integration.txt` if the file was under `multicluster/.coverage` and `.converage`? the doc says it searches project root and current working directory when `-s` is not set.

Reply:
It actually will search current project root directory recursively by default. It's OK to just pass the file name only since we run integration test and MC integration test separately, so it won't have two `coverage-integration.txt` files in one job.

Reply:
Got it